task/bq2bq/executor/bumblebee/transformation.py

```diff
@@ -134,6 +134,27 @@ def transform(self):
                                                             )
 
             transformation.transform()
+        elif self.task_config.load_method is LoadMethod.REPLACE_ALL:
+            # query bq and check if table is partitioned
+            bq_destination_table = self.bigquery_service.get_table(self.task_config.destination_table)
+            if bq_destination_table.time_partitioning is None and bq_destination_table.range_partitioning is None:
+                task_queries = self.sql_query.split(OPTIMUS_QUERY_BREAK_MARKER)
+                transformation = TableTransformation(self.bigquery_service,
+                                                     self.task_config,
+                                                     task_queries[0],
+                                                     self.dstart,
+                                                     self.dend,
+                                                     self.dry_run,
+                                                     localised_execution_time)
+            else:
+                # queries where source data/partition directly map with destination partitions
```

Comment:
I believe the definition is incorrect? as it should be aggregated not directly mapped per partition

Reply:
fixed